feature-configs/demo/sriov/10-sriov-daemonset.yaml

```diff
@@ -0,0 +1,66 @@
+kind: ConfigMap
```

Comment:
Since this is `demo` specific, is it worth checking if what we have on that environment has one of the supported nics? If that's the case. I would remove this.
I am going to check it.